app/models/lot.rb

```diff
@@ -16,5 +16,9 @@ class Lot < ApplicationRecord
 
   # Lots that have a live from that is less than now and a live to that is greater than now
   scope :live, -> { where('live_from <= :t OR (live_from <= :t AND live_to >= :t)', { t: Time.current }) }
-  scope :for_user, ->(user_id) { where('user_id = ?', user_id) }
+  scope :owned_by_user, ->(user_id) { where('user_id = ?', user_id) }
+
+  def live?
+    live_from <= Time.now.utc || (live_from <= Time.now.utc && live_to >= Time.now.utc)
+  end
```

Comment:
```suggestion
  def live?
    live_from <= Time.now.utc || live_to >= Time.now.utc
  end
```
live_from is in the past or live_to is in the future. Means we can turn auctions into timed events with 0 work

Reply:
I'm wrong. The OR would mean lots are always live if live_from is in the past. Don't change